<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wWorkList</class>
 <widget class="QWidget" name="wWorkList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listado de Trabajos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="filterGroupBox">
         <property name="title">
          <string>Filtro</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Buscar:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="searchButton">
              <property name="text">
               <string>Buscar</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/icons/graphics/edit-find.png</normaloff>:/icons/graphics/edit-find.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="clientCheckBox">
              <property name="text">
               <string>Cliente</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="descriptionCheckBox">
              <property name="text">
               <string>Descripcion</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Estado</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QCheckBox" name="recibidoCheckBox">
                 <property name="text">
                  <string>Recibido</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="fotocopiandoCheckBox">
                 <property name="text">
                  <string>Fotocopiando</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="anillandoCheckBox">
                 <property name="text">
                  <string>Anillando</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="terminadoCheckBox">
                 <property name="text">
                  <string>Terminado</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="entregadoCheckBox">
                 <property name="text">
                  <string>Entregado</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="recicladoCheckBox">
                 <property name="text">
                  <string>Reciclado</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Desde:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateTimeEdit" name="desdeDateTimeEdit">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Hasta:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateTimeEdit" name="hastaDateTimeEdit">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="listaCheckBox">
              <property name="text">
               <string>Lista</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
